backend/src/event/create-event.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { IsNotEmpty, IsEnum, IsOptional } from 'class-validator';
+import { EventCategory } from './schemas/event.schema';
+
+export class CreateEventDto {
+  @IsNotEmpty()
+  title: string;
+
+  @IsOptional()
+  description?: string;
+
+  @IsNotEmpty()
+  date: Date;
+
+  @IsNotEmpty()
+  time: string;
+
+  @IsNotEmpty()
```

Comment:
Is that a string or an address ? 

If it's an real address (by autocomplete), you should have longitude and latitude in your schema first. then add in your dto `@IsLatitude()` and `isLongitude()`